packages/block-editor/src/components/inserter/search-items.js

```diff
@@ -116,39 +127,85 @@ export const searchItems = ( items = [], searchTerm = '', config = {} ) => {
 		return items;
 	}
 
-	const defaultGetTitle = ( item ) => item.title;
-	const defaultGetKeywords = ( item ) => item.keywords || [];
-	const defaultGetCategory = ( item ) => item.category;
+	const rankedItems = items
+		.map( ( item ) => {
```

Comment:
Of course no blocker, but seems `reduce` might be better here to do it in one iteration, no?

Reply:
it's probably more performant but I don't think the impact is noticeable and for me, this is more readable.